src/util/codecs.js

```diff
@@ -30,7 +30,7 @@ const getCodecs = function(media) {
 export const isMaat = (master, media) => {
   const mediaAttributes = media.attributes || {};
 
-  return master.mediaGroups.AUDIO &&
+  return master && master.mediaGroups && master.mediaGroups.AUDIO &&
```

Comment:
We should not throw an error if these properties don't exist, not sure if this can happen in the wild.